platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/inv_eeprom.c

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
why not use standard at24 eeprom driver?

Reply:
Guohan
Our hardware implementation cannot use at24 eeprom driver. So we have to use a different eeprom driver. We intend to use this as our implementation for accessing eeprom. We can see if we can use standard AT24 driver and add the change later.